quaireaux/math/src/base64.cairo

```diff
@@ -0,0 +1,225 @@
+mod Decoder {
```

Comment:
I don't think this should be in the `math` crate. Perhaps create an `encoding` crate?

Reply:
I agree with @tarrencev